editor/src/components/canvas/canvas-strategies/strategies/grid-reparent-strategy.tsx

```diff
@@ -273,7 +273,7 @@ function gridReparentCommands(
 
   const gridPropsCommands =
     gridCellGlobalFrames != null && gridTemplate != null
-      ? runGridRearrangeMove(
+      ? runGridMoveRearrange(
```

Comment:
This is probably more like a a followup PR problem, but I think this is a regression, this way reparent to grid  always uses rearrange, so there is no way to just drop an element into a grid without it getting pinned down (which was possible before this PR).

Reply:
indeed, but imo this is something to be done with a followup PR, because we need to decide how to tackle this interaction between blending strategies